crux-rdf/src/crux/rdf.clj

```diff
@@ -208,14 +208,15 @@
        (statements->maps)
        (map #(use-default-language % *default-language*))
        (partition-all tx-size)
-       (reduce (fn [^long n entities]
-                 (when (zero? (long (mod n *ntriples-log-size*)))
-                   (log/debug "submitted" n))
+       (reduce (fn [{:keys [^long entity-count last-tx]} entities]
+                 (when (zero? (long (mod entity-count *ntriples-log-size*)))
+                   (log/debug "submitted" entity-count))
                  (let [tx-ops (vec (for [entity entities]
                                      [:crux.tx/put entity]))]
-                   (db/submit-tx tx-log tx-ops))
-                 (+ n (count entities)))
-               0)))
+                   {:entity-count (+ entity-count (count entities))
+                    :last-tx (db/submit-tx tx-log tx-ops)}))
+
```

Comment:
blank line needed?

Reply:
I tend to, gives more of a visual clue that the `{:entity-count 0}` isn't part of the function?